pilot/pkg/xds/ads.go

```diff
@@ -282,7 +281,7 @@ func (s *DiscoveryServer) Stream(stream DiscoveryStream) error {
 		// Error accessing the data - log and close, maybe a different pilot replica
 		// has more luck
 		adsLog.Warnf("Error reading config %v", err)
-		return err
+		return status.Error(codes.Unavailable, "error reading config")
```

Comment:
do you want to include err in the message?

Reply:
yes, for some reason I was thinking we didn't currently, but I just read it wrong. will update